### ISSUE TITLE
Brake Temp + Engine Master + CPDLC + FCU SPD Issues

### ISSUE THREAD

**Issue by MoonFlare320** [D1ngtalk/Yourcontrols-config-for-Fenixsim-A320#9]
Hi there,

We encountered four further (!minor!) issues when flying:

1. The Brake-Temperature issue seems to be resolved now, however, in the MCDU-Maint-Active Failures Menu, for both of the pilots, the "High Break Temperature Failure" still is indicated and has to be cleared.
2. When performing a Single-Engine Taxi-In, and one of the two pilots (it doesn't matter who exactly, from what we found out) turns off the Engine 2 Master, it does not sync for the other pilot.
3. CPDLC and PDC Messages only arrive for one of the two pilots, it seems to be random which one. --> I guess that isn't really to be fixed, but rather an issue with Hoppie and two connections with the same call sign, but perhaps there is any data ref to just read out the received DCDU messages?
4. If the winds aren't 100% the same and one of the two pilots "pulls" the Speed to go into selected Speed Mode, the speeds can differ for the two pilots, which then results in a difference regarding the speed, which can be quite significant.

As said, this file, in my opinion, already is an absolute game changer. All of these things are very minor, but still, it would be absolutely superb if even these little things could also be fixed.

Thanks a lot and best regards from Austria!

**Comment by hurfifx** [comment]
Hey for the number 4 did you make sure you were both on the same server in msfs?

**Comment by MoonFlare320** [comment]
Yes, however, a small difference still exists and has an effect accordingly.

**Comment by hurfifx** [comment]
> Yes, however, a small difference still exists and has an effect accordingly.

okay maybe a problem on asobo side, like due to your location from the server (maybe latency ms)

**Comment by MoonFlare320** [comment]
@D1ngtalk Could you perhaps have a look regarding 1. and 2.? 

I guess 2. could be a rather easy fix :)

Thanks a lot!
